src/common/copilot/PowerPagesCopilot.ts

```diff
@@ -144,7 +144,7 @@ export class PowerPagesCopilot implements vscode.WebviewViewProvider {
     ) {
         this._view = webviewView;
 
-        webviewView.title = "Copilot In Power Pages" + (IS_DESKTOP ? "" : " [PREVIEW]");
+        webviewView.title = vscode.l10n.t("Copilot In Power Pages") + (IS_DESKTOP ? "" : " [PREVIEW]");
```

Comment:
why are we not showing PREVIEW in Desktop?

Reply:
We do show it but using the 'description' property, which is not available in case of vscode.dev and hence static string is req.

![image](https://github.com/microsoft/powerplatform-vscode/assets/54068463/5cec5094-e888-4991-b29c-709d67f9e08a)

Reply:
aha- forgot this one - better leave a comment on this one next time in the PR code